src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java

```diff
@@ -67,7 +67,7 @@ public HealthCheckResource() throws Exception {
             String clientConfigFilePath;
             while ((clientConfigFilePath = reader.readLine()) != null) {
                 lightblueClients.put(
-                        clientConfigFilePath.replace(".properties", ""),
+                        new String(clientConfigFilePath).replace(".properties", ""),
```

Comment:
Not a big deal, but why the new String? I didn't think the replace method was destructive.